runtime/src/cost_model.rs

```diff
@@ -200,6 +206,59 @@ impl CostModel {
         }
     }
 
+    fn calculate_account_data_size_on_deserialized_system_instruction(
+        instruction: SystemInstruction,
+    ) -> u64 {
+        match instruction {
+            SystemInstruction::CreateAccount {
+                lamports: _lamports,
+                space,
+                owner: _owner,
+            } => space,
+            SystemInstruction::CreateAccountWithSeed {
+                base: _base,
+                seed: _seed,
+                lamports: _lamports,
+                space,
+                owner: _owner,
+            } => space,
+            SystemInstruction::Allocate { space } => space,
+            SystemInstruction::AllocateWithSeed {
+                base: _base,
+                seed: _seed,
+                space,
+                owner: _owner,
+            } => space,
+            _ => 0,
```

Comment:
How often do new System Instructions get added? If the answer is "never", then this is fine. If new system instructions do get added, maybe it'd be worthwhile to _not_ have the catch-all `_` case, and instead list every instruction. This would enforce that new instructions are evaluated for their impact on the account data.

Reply:
This is a good idea

Reply:
done